sympy/combinatorics/perm_groups.py

```diff
@@ -155,6 +155,7 @@ def __new__(cls, *args, **kwargs):
         obj._is_trivial = None
         obj._transitivity_degree = None
         obj._max_div = None
+        obj._is_polycyclic = None
```

Comment:
I think this can be omitted. It is just a copy of `_is_solvable`.

Reply:
okay